### ISSUE TITLE
Fetching events fails with 0.3.6+

### ISSUE THREAD

**Issue by luckymike** [miasma-rb/miasma-aws#44]
When a stack has many events, fetching them fails with version `0.3.6+` (using `sfn`). I've isolated the issue to [this line](https://github.com/miasma-rb/miasma-aws/blob/master/lib/miasma/contrib/aws/orchestration.rb#L406):

```
stack.custom[:last_event_token] = event['NextToken'] if event['NextToken']
```

See also sparkleformation/sfn#266

**Comment by gramosbr** [comment]
If I'm reading this correctly stack.custom is frozen by 
```
attribute :custom, Smash, :coerce => lambda{|v| v.to_smash}, :default => Smash.new
```
https://github.com/miasma-rb/miasma/blob/51fd736f123e3684ee34ad4727711c7cbae7ebc2/lib/miasma/types/model.rb#L12-L12

**Comment by luckymike** [comment]
The code in #45 is working for me.

**Comment by chrisroberts** [comment]
Was fixed in #45
